ci/docker-db.sh

```diff
@@ -1,59 +1,76 @@
 #!/usr/bin/env bash
-# source this file to setup postgres and mysql
-# for local testing (as similar as possible to docker)
+# This scripts starts a postgres or mysql database server, waits for it, and
+# setup setups users to be used for local testing.
 
 set -eu
 
-export MYSQL_HOST=127.0.0.1
+# These environment variables influence the mysql and psql clients
+export MYSQL_HOST=${MYSQL_HOST:-127.0.0.1}
 export MYSQL_TCP_PORT=${MYSQL_TCP_PORT:-13306}
-export PGHOST=127.0.0.1
-NAME="hub-test-$DB"
-DOCKER_RUN="docker run -d --name $NAME"
-
-docker rm -f "$NAME" 2>/dev/null || true
-
-case "$DB" in
```

Comment:
Is there a reason we can't use a switch statement anymore?

Reply:
It was the readability for myself with some motivation that I believed it could be improved also for others.